ansible/roles/nomis-weblogic/templates/10.3/etc/init.d/weblogic-healthcheck

```diff
@@ -1,45 +1,90 @@
 #!/bin/bash
 #
 # weblogic-healthcheck        Control weblogic healthcheck file
-#
-# chkconfig: - 95 05
-# description: weblogic server
 
```

Comment:
chkconfig and description lines need to be there.  It controls order things start up.  Fix description though.